ducktape/cluster/remoteaccount.py

```diff
@@ -62,6 +63,8 @@ def _can_ping_url(self, url, headers):
     def ssh_command(self, cmd):
         if self.local:
             return cmd
+        if self.sudo_user is not None:
```

Comment:
@granders Here are my thoughts:

- Adding it in RemoteAccount is just convenient if we believe the same sudo_user will be used for all ssh_command from this RemoteAccount instance. It is similar to user which is put in RemoteAccount instead of in ssh_command(). It is not necessary and either way works for my use case.
- In fact, the use case here is that we don't want to ssh in as root but we may need to start/stop process as root, which is already demonstrated in your use case. I agree that you can get around by doing `node.account.ssh("sudo cmd")`. But what if I want to do `node.account.kill_process(...)` to kill process as root user?

Simply speaking, the use case is to be able to execute command on remote machine as root user without logging into that machine as root user, using APIs already provided in RemoteAccount, including ssh_command() and kill_process(). Is this reasonable?

I think the solution to the use case above is to be able to specify sudo_user in either function parameter or in the RemoteAccount constructor. Is there better solution to this problem?